bentoml/cli/bento.py

```diff
@@ -237,3 +245,112 @@ def retrieve(bento, target_dir):
         safe_retrieve(bento_service_bundle_path, target_dir)
 
         click.echo('Service %s artifact directory => %s' % (name, target_dir))
+
+    @cli.command(
```

Comment:
Maybe we should rename the command to `create-docker-image` using the `name=`? I want something very simple and direct.  What do you think?

Reply:
IMO `containerize` is pretty self-explanatory and can be left as-is. Thoughts, @parano?

Reply:
@jackyzha0 yea I think `containerize` works